### ISSUE TITLE
bugtool: get the access log 

### ISSUE THREAD

**Issue by aalemayhu** [cilium/cilium#2402]
Requires an API change. I am not sure we want to do this but want to move this item out of #2165. So if it's not appropriate, this issue can be closed.

**Comment by manalibhutiyani** [comment]
Marking this as postponed till the api change is in place. Assigning to you for the time being. Please close / re-assign if needed. 

This issue is currently blocked on providing an api for getting access log.

**Comment by aalemayhu** [comment]
Sounds good.
